kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/json/JSONMarshaller.java

```diff
@@ -347,6 +356,17 @@ public String marshall(Object objectInput) {
         }
     }
 
+    @Override
+    public byte[] marshallAsBytes(Object objectInput) {
```

Comment:
why is this required ?

Reply:
this is used in RawJsonWriter. It is not strictly needed, but save a byte[]-string conversion, which provide some peformance gain

Reply:
move to the marshaller interface with default implementation UnsupportedOperationException

Reply:
it is in the marshaller with default implementation to call the string one